### DISCUSSION TITLE
유니티 상에서는 게임동작이 되는데, 모바일 구동 시 에셋이 사라집니다.

### DISCUSSION THREAD

**Discussion by BumBall-Maker** [naverz/zepeto-studio-kor#945] [5. Q&A]
유니티 상에서는 멀티플레이랑 게임씬에서 아바타 다 보이고 구동이 되는데
QR 내보내기를 한 후 모바일로 월드를 보면 만들어둔 건축물들이 사라지고 바닥만 보이게 됩니다.
어떤 이슈인지 감이 안잡히네요 
![image](https://user-images.githubusercontent.com/87287709/176848109-415fc831-9f0b-4e95-bf9b-3903d6ccb282.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 테스트에서 사용하신 모바일 기종이 어떻게 되시나요?
모바일에서 바닥이 보인다는 것은 터레인의 지형과 바닥 메테리얼은 보이는데
건물 어셋만 보이지 않는다는 것인지도 궁금합니다.
이슈 파악을 위해 기본적으로 월드 버전, 개발 환경 등 상세 정보 기재를 부탁 드립니다.

**Reply by BumBall-Maker** [reply]
![image](https://user-images.githubusercontent.com/87287709/176887714-ed51181b-480a-4d80-aa50-26ea5c2cf221.png)
월드 버전 : 2020.3.36f1
개발 환경 : window, unity
터레인 지형이나 바닥 머테리얼도 보이지 않는 상태입니다

**Reply by BumBall-Maker** [reply]
멀티플레이 관련 문제같습니다
하이어라키창과 스크립트 제거 하니 구동이 되었습니다
멀티플레이 관련 문제 해결 도와주시면 감사하겠습니다

**Reply by jehkael** [reply]
이 문제 어떻게 해결하셨나요?
